validation/clients/openssl/client.c

```diff
@@ -15,9 +15,57 @@
 #include <openssl/ssl.h>
 #include <openssl/x509.h>
 #include <openssl/x509_vfy.h>
+#include <openssl/ocsp.h>
 
 #include "client.h"
 
+/* Taken from "OpenSSL_1_1_1-stable/apps/apps.c" with minor changes. */
+static int load_cert_crl_http(const char *url, X509 **pcert, X509_CRL **pcrl)
```

Comment:
This function could be rewritten to be more concise:

- `pcert` is always NULL, it doesn't need to be there
- `use_ssl` will always be false in our case

Reply:
That's true, but it does not make the code much less readable, so I'd keep the current version.